Library/Homebrew/extend/os/mac/extend/ENV/shared.rb

```diff
@@ -26,9 +26,12 @@ def no_weak_imports_support?
 
   sig { returns(T::Boolean) }
   def no_fixup_chains_support?
-    return false if !MacOS::CLT.version.null? && MacOS::CLT.version < "13.0"
-    return false if !MacOS::Xcode.version.null? && MacOS::Xcode.version < "13.0"
+    ld_v = Utils.safe_popen_read("/usr/bin/ld", "-v", err: :out).lines.first.chomp
```

Comment:
Do we still get that xcrun stderr noise on I think 11-arm64?

Reply:
There is `-version_details` for better parsing (it's JSON) though it requires Xcode 10.2+.

Though with that said we could hardwire < Mojave to false which covers < 10.2.

Reply:
I think we still see a lot of xcrun stderr noise. Used `-version_details` instead, so we don't accidentally catch any stderr garbage.